cvat-ui/src/reducers/annotation-reducer.ts

```diff
@@ -55,6 +57,11 @@ const defaultState: AnnotationState = {
         },
         playing: false,
         frameAngles: [],
+        contextImage: {
+            loaded: false,
+            data: '',
```

Comment:
Do we need loaded flag? Could we set data to ``null`` meaning that image was not loaded?

Reply:
Yes we require the loaded flag to ensure we show the contextImage during load only and also used to trigger action on pcd change.